source/matrix_free/matrix_free.inst.in

```diff
@@ -82,7 +82,6 @@ for (deal_II_dimension : DIMENSIONS)
       const std::vector<hp::QCollection<1>> &,
       const AdditionalData &);
 
-#ifndef DEAL_II_MSVC
```

Comment:
Nice. I think that with as many problems as MSVC has given us over the years, we can assume that we are always only supporting the most recent version we can get our hands on...